lib/modules/manager/helmfile/extract.spec.ts

```diff
@@ -363,6 +363,36 @@ describe('modules/manager/helmfile/extract', () => {
       });
     });
 
+    it('parses a chart with an oci repository with ---', async () => {
+      const content =
+        'repositories:\n' +
+        '  - name: oci-repo\n' +
+        '    url: ghcr.io/example/oci-repo\n' +
+        '    oci: true\n' +
+        '---\n' +
+        'releases:\n' +
+        '  - name: example\n' +
+        '    version: 0.1.0\n' +
+        '    chart: oci-repo/example';
```

Comment:
use a codeBlock instead
https://github.com/renovatebot/renovate/blob/e2a7586466df0370ac71659332b5fb459a483d4b/lib/util/toml.spec.ts#L6-L10

Reply:
eh, last time i tried that, it failed on indentations :/ ill try